frontend/ui/detailview.jsx

```diff
@@ -420,6 +428,36 @@ class DetailViewComponent extends React.Component {
   }
 
   render() {
+    const legends = this.getLegend().sort((a, b) => a.title < b.title);
+    const {
+      measure,
+      platform,
+      timeInterval
+    } = this.state;
+    let crashStatsLink;
+    if (measure in MEASUREMENTS) {
+      const { processType, extraParams } = MEASUREMENTS[measure];
+      const now = Date.now();
+      let queryParams;
```

Comment:
I believe you should be able to get this variable in a one liner, instead of building it up:

```
const queryParams = stringify({...}) + extraParams ?  `${stringify(Object.keys(extraParams)
+          .reduce((dict, key) => ({ ...dict, [key]: extraParams[key] }), {}))}&` : '';
```

Reply:
I don't think that works.

In any case, I should another approach in my next push.